src/peft/tuners/lora.py

```diff
@@ -72,7 +86,7 @@ def __post_init__(self):
         self.peft_type = PeftType.LORA
 
 
-class LoraModel(torch.nn.Module):
+class LoraModel(PushToHubMixin, torch.nn.Module):
```

Comment:
is there any specific advantage to using `transformers.PushToHubMixin` vs `huggingface_hub`'s? the latter means you only have to implement a single `_save_pretrained()` method, but perhaps you need more flexibility

Reply:
I was not aware that the class you propose is simpler, I will have a look now and potentially replace it !

Reply:
I tried it and in both cases I needed to re-implement `from_pretrained` method, also it seems that `push_to_hub` from `PushToHubMixin` first clones a repo with the weights before pushing on the Hub, maybe I'll just stick on `transformers.PushToHubMixin`.